settings.json.template

```diff
@@ -409,6 +409,12 @@
   "indentationOnNewLine": false,
   */
 
+  /*
+   * If true, importing to a pad is allowed only if an author has a session
+   * estabilished and has already contributed to that specific pad.
+   */
```

Comment:
I'd really like an help to better describe what happens when this setting is true (use case, only applies to imports via the web interface? Has an effect on API calls?).

Depending on this "brainstorming", we may also want to:
- how to describe this change in the docs (we can include this in #3858)?
- do we want to flip it on when an old installation upgrades to **1.8.3** (see my next comment)?

Reply:
Done